typegen/src/tests/utils.rs

```diff
@@ -37,7 +37,7 @@ impl Testgen {
 
     pub fn gen(self, settings: TypeGeneratorSettings) -> TokenStream {
         let mut registry: PortableRegistry = self.registry.into();
-        ensure_unique_type_paths(&mut registry);
+        ensure_unique_type_paths(&mut registry).expect("corrupted PortableRegistry");
```

Comment:
Can you be consistent whether to use upper/lowercase of `Corrupted or corrupted` in the expect?

Reply:
done